modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/product/navigation/personal/menu/MyDashboardPersonalMenuEntry.java

```diff
@@ -97,7 +100,10 @@ public boolean isShow(
 			PortletRequest portletRequest, PermissionChecker permissionChecker)
 		throws PortalException {
 
-		if (!PropsValues.LAYOUT_USER_PRIVATE_LAYOUTS_ENABLED) {
+		if (!PrefsPropsUtil.getBoolean(
+				CompanyThreadLocal.getCompanyId(),
```

Comment:
@torresvictor100, if you face issues with this `CompanyThreadLocal.getCompanyId`, you can try to use the method `PortalUtil.getCompanyId(portletRequest)`